mobile/src/main/res/values/strings.xml

```diff
@@ -121,5 +121,16 @@
     <string name="notification_posted">Notification posted</string>
     <string name="pref_default_maximum_retries">1</string>
     <string name="pref_title_maximum_retries">Maximum Retries</string>
+    <string name="action_about">About</string>
+    <string name="action_home_mycroft_ai">Home.Mycroft.Ai</string>
```

Comment:
How about Mycroft Management Website, I think of http://mycroft.ai  as the mycroft homepage.